src/systemtest/resources/author-config.csv

```diff
@@ -1,5 +1,5 @@
 Repository's Location,Branch,Author's GitHub ID,Author's Email,Author's Display Name,Author's Git Author Name,Ignore Glob List
-https://github.com/reposense/testrepo-Beta.git,,nbr,neilbrian.nl@gmail.com,"Ashwin Mukadaan Singhania, Rathod",,
+https://github.com/reposense/testrepo-Beta.git,,nbr,neilbrian.nl@gmail.com,"Ashwin Mukadaan Singhania, Rathod",nbriannl,
```

Comment:
@jylee-git I updated the `author-config.csv` used for the system tests.
Reason: Within `testRepo-Beta`, some files contain author tags within the name `nbriannl` (i.e. `@@author nbriannl`).
However, none of the authors in `author-config.csv` have the alias `nbriannl`.

From an inspection of the standalone config in testRepo-Beta ([here](https://github.com/reposense/testrepo-Beta/blob/master/_reposense/config.json)), it seems that the `nbriannl` alias belongs to the author whose github id is `nbr` in the `author-config`.

I updated the aliases list for the author `nbr` in `author-config`, so that `AnnotatorAnalyzer` will attribute code to him / her in the system tests.

Reply:
Alright, makes sense.

Reply:
I don't think this is in the scope of this PR.

Since in the `repo-config.csv` of the systemtest, notice that the standalone config file is ignored. So i'm of the opinion that we should leave it as it isl

Reply:
Sure, I've removed the changes to the `author config csv`